noxfile.py

```diff
@@ -197,76 +204,78 @@ def tests(session: nox.sessions.Session):
     )
 
 
-@nox.session(python=PY_VER, venv_backend="conda")
-def gallery(session: nox.sessions.Session):
-    """
-    Perform iris gallery doc-tests.
+if PY_VER == _PY_VERSION_DOCSBUILD or _PY_VERSION_DOCSBUILD in PY_VER:
```

Comment:
I can see the difference that this line makes, but I can't see that it actually _gives_ us anything. We have enough security in the `@nox.session(python=` kwarg that Nox will only run the given sessions with the correct python version. 

Indeed your changes mean we can instead drop the `PY_VER` setting from the relevant Cirrus runs.

Reply:
It's only for local runs :  because **_if_** you say "PY_VER=3.7 nox -s doctests", it will run under 3.8, which seems unexpected and presumably not what was specifically intended.

Reply:
>because if you set "PY_VER=3.7" and run "nox -s doctests", it will run under 3.8 regardless

This is no longer true thanks to you using `python=_PY_VERSION_DOCSBUILD` in the relevant decorators.

Reply:
> thanks to you using python=_PY_VERSION_DOCSBUILD in the relevant decorators.

? but [as currently configured](https://github.com/SciTools/iris/blob/28d1472417ea05bd5015a6f66e3e35085c1ad9ad/noxfile.py#L19-L23), _PY_VERSION_DOCSBUILD  is _always_ == 3.8

So surely it _does_ run under 3.8, if it runs at all.
( The aim here was to stop it running at all, if asked for 3.7 )

Reply:
> So surely it does run under 3.8, if it runs at all.

Yes, precisely. But that is already achieved _without_ generating the relevant sessions within a conditional block. IMO the extra convenience of the conditional block isn't worth the extra complexity / potential confusion.

Reply:
Ok, I'll simplify.